panel/panel.py

```diff
@@ -136,64 +119,51 @@ def run_update_panel_cmd(panel, text=None):
     panel.run_command(cmd, {'text': text, 'clear_sel': clear_sel})
 
 
-def format_row(lineno, error_type, dic):
-    lineno = int(lineno) + 1
-    start = dic['start'] + 1
-    msg = dic['msg'].rstrip()
```

Comment:
Where is the `rstrip`?

Reply:
In Vegas? Let's place them upstream in the data flow like in `Linter.error` bc status bar messages would profit from it as well. And why not `strip` both sides altogether?